src/commands/CreateCommand.js

```diff
@@ -0,0 +1,33 @@
+import Command from "../Command";
+import ConfigUtilities from "../ConfigUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import NpmUtilities from "../NpmUtilities";
+import path from "path";
+
+export default class LsCommand extends Command {
+  initialize(callback) {
+    this.pkgName = this.input[0];
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    const author = NpmUtilities.getWhoIAm();
+    const name = this.pkgName;
+    const pkgRoot = path.join(".", "packages", this.pkgName);
```

Comment:
This needs to support the configurable `packages` location(s), yes?

(presumably picking the first location in the event multiple exist)